docs/organizations/managing-security-and-risk.md

```diff
@@ -131,6 +131,13 @@ The following section details when Codacy opens and closes items for each suppor
         -   If an issue is opened and closed on the same day, Codacy may not detect it.
         -   To make sure that Codacy detects Jira issues correctly, assign the "security" label when creating the issue or immediately after.
 
+## Exporting items
```

Comment:
@nicklem I find it weird to have a dedicated section for this (small) functionality instead of referring to the export capability directly in the **Dashboard** and **Item list** sections... 🤔 

Is there a technical limitation behind it? I'm asking this also following the admonition note: the **Export CSV** button generates the same list of items from the dashboard or the items list?

Reply:
I chose to use this format to add to the list of tasks that the user can do and answer the question "How can I export a list of items?".

It's a UX consideration and also there to avoid repeating the instruction. There's no real technical limitation.

Thoughts?

Reply:
Don't consider my comment a blocker. I see your point and it's good to go like it is. 🚀 

I think it's the UI that's confusing me, we can go over this in our peer review. 👍